CMakeLists.txt

```
@@ -11,6 +11,3 @@ option(DAWN_BUILD_TESTING "Enable Dawn tests." ${BUILD_TESTING})
 option(DAWN_BUILD_EXAMPLES "Enable Dawn examples." ${BUILD_TESTING})
 add_subdirectory(dawn)
 
```

```suggestion
message(STATUS "Configuring GTClang...")
option(GTCLANG_BUILD_TESTING "Enable GTClang tests." ${BUILD_TESTING})
add_subdirectory(gtclang)
```

needs to be added agian, deactivated to be able to build on daint